Packages/com.unity.inputsystem/InputSystem/InputUpdateType.cs

```diff
@@ -16,32 +16,29 @@ public enum InputUpdateType
 
         /// <summary>
         /// Update corresponding to <a href="https://docs.unity3d.com/ScriptReference/MonoBehaviour.Update.html">Update</a>.
-        /// </summary>
-        /// <remarks>
+        ///
         /// Every frame has exactly one dynamic update. If not reconfigured using <see cref="PlayerLoop"/>,
         /// the dynamic update happens after all the fixed updates for the frame have run (which can be
         /// zero or more).
         ///
         /// Input updates run before script callbacks on MonoBehaviours are fired.
-        /// </remarks>
+        /// </summary>
```

Comment:
`<remarks>` is not actually supported on enum values by docfx. Will just silently get swallowed (just like `<seealso`).